multiprocessing_parse.py

```diff
@@ -185,5 +210,6 @@ def readText(files_in_directory):
                                                         timeout=20)
         if not rawVectorData:
             raise TypeError("\nNo vectors created")
-
-        return np.array(rawVectorData)
+        rawVectorData = np.array(rawVectorData, dtype=np.float32)
+        assert rawVectorData.dtype == np.float32
```

Comment:
Will this give readable error? I would rather use raise as it is meant to do that.

Reply:
Yes it will raise assertion error